src/webhook/validation/oneagent.go

```diff
@@ -90,3 +95,47 @@ func hasConflictingMatchLabels(labelMap, otherLabelMap map[string]string) bool {
 	otherLabelSelectorLabels := labels.Set(otherLabelMap)
 	return labelSelector.Matches(otherLabelSelectorLabels) || otherLabelSelector.Matches(labelSelectorLabels)
 }
+
+func getOneAgentVolumeStorageEnvVar(env []corev1.EnvVar) (bool, bool) {
```

Comment:
why the 2 bools ???

we only care if its explicitly disabled (AFAIK)

Reply:
For the warning in classicFullstack mode it's necessary to know if the flag was set. The value is rather irrelevant in this use-case.

Reply:
```suggestion
func getOneAgentVolumeStorageEnvVar(env []corev1.EnvVar) (enabled bool,  set bool) {
```
maybe for better readability 🤔